### ISSUE TITLE
Visibility of     `Example Value |    Model` tabs bad

### ISSUE THREAD

**Issue by dothebart** [swagger-api/swagger-ui#4239]
On firefox / linux the pipe gets mixed into the text:
![screenshot_model](https://user-images.githubusercontent.com/5013077/36421817-3c7c9a54-163a-11e8-8323-09f99e401811.png)
![screenshot_example](https://user-images.githubusercontent.com/5013077/36421825-4030d70a-163a-11e8-852f-dd6a55b42fe1.png)

But even on macos its not really visible that these are tabs which the user can toggle. 
Please improve the visibility of these elements.

**Comment by heldersepu** [comment]
There was a lot of good info that you skipped on the issue template like:

| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  
| Which Swagger/OpenAPI version?  |
| Which Swagger-UI version?       |
| How did you install Swagger-UI? | 
| Which browser & version?        |
| Which operating system?         |

**Comment by dothebart** [comment]
its both. Its outright broken in firefox 52.6.0 (see the screenshots, but I'd consider that minor) and the useability is not good which would be a feature request to improve this (which is rather major). 
Since I don't think that from what the UI presents users will see that it can be clicked to toggle this.

My swagger json (if that helps) https://gist.github.com/dothebart/7c2fd526698037c73d434677b9f37329

We updated Swagger-UI 14 days ago, so whatever was current then.

**Comment by heldersepu** [comment]
Update your initial post with the template if you can, that makes it easier for everyone...

**Comment by heldersepu** [comment]
It looks good to me on: 
- Chrome(Version 64.0.3282.167 (Official Build) (64-bit)) 
- Firefox(58.0.2 (64-bit))

I am using windows 10 but I doubt that the OS is the problem

<br>
Take a look at your swagger json with the latest ui:

http://petstore.swagger.io/?url=https://gist.githubusercontent.com/dothebart/7c2fd526698037c73d434677b9f37329/raw/32162253075eaafdde898f61f054b33bf482684b/api-docs.json#/Cluster/post__admin_cluster_test

The [validator ](https://online.swagger.io/validator/debug?url=https://gist.githubusercontent.com/dothebart/7c2fd526698037c73d434677b9f37329/raw/32162253075eaafdde898f61f054b33bf482684b/api-docs.json)shows a bunch of issues...

**Comment by dothebart** [comment]
In the part of `/_api/aqlfunction Return registered AQL user functions` the `Description` column changes its size when one clicks back & forth [Example Value | Model] ? I see that in Chromium and Firefox.

**Comment by heldersepu** [comment]
@dothebart Yes it changes size maybe you want to report that in a new issue ...
And please fill the issue template

**Comment by dothebart** [comment]
ok. But please keep this as a feature request to make this tab switching mechanism better visible in the UI. 
I've put a Collegue in front of the UI and told him to *search* where he would click (to verify my claim), and it took him 5 minutes to find that one could click to switch the tab content.

**Comment by dothebart** [comment]
ok, adding a screenshot to our swagger blogpost https://twitter.com/arangodb/status/976868182047764481 so at least our users can find it ;-)
